Scenes/World/Background.gd

```diff
@@ -0,0 +1,31 @@
+extends Node2D
+
+export(int) var num_of_layers = 1
+export(float) var move_speed
+export(float) var tree_seperation
+
+var Layer = load("res://Objects/Background/Layer.tscn")
+
+func _ready() -> void:
```

Comment:
It will be easier to read if what is in `_ready` will be done as a separate two func